frontend/containers/DocumentList/DocumentList.jsx

```diff
@@ -168,6 +195,28 @@ class DocumentList extends Component {
     actions.clearDocumentList()
   }
 
+  renderField(fieldName, schema, value) {
```

Comment:
Or not! It could be a file written by webpack at build time or something. I can have a play with this as I keep working on document edit.